user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_alarms.md

```diff
@@ -5,3 +5,6 @@ uid: Get_alarms
 # Get alarms
 
 Available from DataMiner 10.2.0/10.1.9 onwards. The *Get alarms* data source retrieves the alarms in the DataMiner System. Several columns, such as *Element Name*, *Parameter Description*, *Value* and *Time*, are included by default. Others can be added with a [*Select* operation](xref:GQI_Select).
+
+> [!NOTE]
+> If a Cassandra database is used, we recommend always adding filtering on *Time*, as this filters directly on the database. Otherwise, the SLDataGateway process will fetch all alarms and then apply the filtering, which might take a lot of resources.
```

Comment:
Does this only apply for self-hosted setups with Cassandra, or also for STaaS?

Reply:
Yes, LGTM!